app/routes/overview/components/PublicFrontpage.tsx

```diff
@@ -1,256 +1,187 @@
 };
 
-class PublicFrontpage extends Component<Props, State> {
-  state = {
-    registerUser: false,
-    forgotPassword: false,
-  };
-  toggleRegisterUser = () =>
-    this.setState({
-      registerUser: true,
-    });
-  toggleForgotPassword = () =>
-    this.setState({
-      forgotPassword: true,
-    });
-  toggleBack = () =>
-    this.setState({
-      registerUser: false,
-      forgotPassword: false,
-    });
+const isEvent = (
+  item: WithDocumentType<PublicArticle> | WithDocumentType<PublicEvent>
+): item is WithDocumentType<PublicEvent> => item.documentType === 'event';
+
+const isArticle = (
+  item: WithDocumentType<PublicArticle> | WithDocumentType<PublicEvent>
+): item is WithDocumentType<PublicArticle> => item.documentType === 'article';
 
-  render() {
-    const { registerUser, forgotPassword } = this.state;
+const PublicFrontpage = ({ frontpage, readmes }: Props) => {
+  return (
+    <Container>
+      {/* <Banner
+           header="Abakusrevyen har opptak!"
+           subHeader="Søk her"
+           link="https://opptak.abakus.no"
+           color={COLORS.red}
+          /> */}
+      <Container className={styles.container}>
+        <Card style={{ gridArea: 'welcome' }}>
+          <Welcome />
+        </Card>
+        <Card style={{ gridArea: 'login' }}>
+          <AuthSection />
+        </Card>
+        <Card style={{ gridArea: 'events' }}>
+          <CompactEvents events={frontpage.filter(isEvent)} frontpageHeading />
+        </Card>
+        <Card style={{ gridArea: 'hsp' }}>
+          <HspInfo />
+        </Card>
+        <Card style={{ gridArea: 'article' }}>
+          <LatestArticle frontpage={frontpage} />
+        </Card>
+        <LatestReadme
+          readmes={readmes}
+          expandedInitially
+          collapsible={false}
+          style={{ gridArea: 'readme' }}
+        />
+        <Card style={{ gridArea: 'links' }}>
+          <UsefulLinks />
+        </Card>
+        <Card style={{ gridArea: 'facebook' }}>
+          <Facebook />
+        </Card>
+      </Container>
+    </Container>
+  );
+};
 
-    const isEvent = (item) => item.documentType === 'event';
+const Welcome = () => (
+  <>
+    <h2 className="u-mb">Velkommen til Abakus</h2>
+    <p>
+      Abakus er linjeforeningen for studentene ved <i>Datateknologi</i> og
+      <i> Kommunikasjonsteknologi og digital sikkerhet</i> på NTNU, og drives av
+      studenter ved disse studiene.
+    </p>
+    <p>
+      Abakus{"'"} formål er å gi disse studentene veiledning i
+      studiesituasjonen, arrangere kurs som utfyller fagtilbudet ved NTNU,
+      fremme kontakten med næringslivet og bidra med sosiale aktiviteter.
+    </p>
+  </>
+);
 
-    const isArticle = (item) => item.documentType === 'article';
+const HspInfo = () => (
+  <div className={styles.hsp}>
+    <h3>
+      <a href="https://www.netcompany.com/no" target="blank">
+        <Image className={styles.hspImage} src={netcompany} alt="NETCOMPANY" />
+      </a>
+    </h3>
```

Comment:
I'll write it up as a todo that we should support a dark theme hsp logo, because this one is very difficult to read on dark theme.

Also, why is this inside of an `h3` header? For universal accessibility?

Reply:
Nice.
The `h3` tag was mostly to get some padding, but I suppose it helps for accessibility as well:)